recipes/thrax/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "thrax" %}
+{% set version = "1.3.1" %}
+
+package: 
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.opengrm.org/twiki/pub/GRM/ThraxDownload/{{ name }}-{{ version }}.tar.gz
+  sha256: d16a24376c8e722b694825366eda8a8accfd7515abb7a3db563c19056f951203
+  
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - openfst ==1.7.5
+  run:
+    - openfst ==1.7.5
+    - python
+
+test:
+  requires:
+    - openfst ==1.7.5
+  commands:
+    - thraxcompiler --helpshort >/dev/null || [[ $? == 1 ]]
```

Comment:
Same questions about redirection and exit codes

Reply:
Good point, fixed.